<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="nombreTienda">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Servicio Técnico de Móviles</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="usuario">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="conectarServidor">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>671</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTiendas">
     <attribute name="title">
      <string>Tiendas</string>
     </attribute>
     <widget class="QComboBox" name="tiendasCmBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>331</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tiendaNombre">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tiendaUser">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tiendaPassword">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tiendaDireccion">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Direccion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tiendaCrear">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tiendaModificar">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tiendaEliminar">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTecnicos">
     <attribute name="title">
      <string>Técnicos</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QComboBox" name="tecnicoCmBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>331</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tecnicoNombre">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tecnicoEliminar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>210</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tecnicoModificar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>150</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>MOdificar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tecnicoPassword">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tecnicoCrear">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>90</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tecnicoUser">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMarcas">
     <attribute name="title">
      <string>Marcas</string>
     </attribute>
     <widget class="QPushButton" name="marcaCrear">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="marcaModificar">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>170</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>MOdificar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="marcasNombre">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="marcaEliminar">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>230</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QComboBox" name="marcasCmBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>331</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabModelos">
     <attribute name="title">
      <string>Modelos</string>
     </attribute>
     <widget class="QPushButton" name="modelosCrear">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>190</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modelosModificar">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>240</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modeloNombre">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modelosEliminar">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>290</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QComboBox" name="modeloMarcaCmBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>241</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="modeloCmBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>241</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Marcas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Modelos</string>
      </property>
     </widget>
     <widget class="QListWidget" name="modeloReparacionesPosibles">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="modeloReparacionesElegidas">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>161</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Reparaciones disponibles</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Reparaciones elegidas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabReparaciones">
     <attribute name="title">
      <string>Reparaciones</string>
     </attribute>
     <widget class="QPushButton" name="reparacionCrear">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reparacionModificar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>160</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>MOdificar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reparacionNombre">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reparacionEliminar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>220</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QComboBox" name="reparacionesCmBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>331</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
